guide/daily-spending-wallet/backup-and-recovery/landing-page.md

```diff
@@ -52,32 +74,27 @@ Two main types of information need to be backed up: the user's wallet recovery d
    height = 453
 %}
 
-- The _[Recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }})_ is the fundamental information (consisting of 12 or 24 words) used to construct a wallet
-- The _Passphrase_ is an optional password for additional security
-- The _[Derivation path]({{ '/guide/glossary/#derivation-path' | relative_url }})_ includes information on how to construct the wallet
-- A _Fingerprint_ allows for verification that the wallet was reconstructed correctly after import
-- _Static channel backups_ contain all basic information about active Lightning channels
-- A _Channels database_ contains data on payments made in active Lightning channels
+### Recovery Phrase
 
-Users who lose both their wallet access and their backup permanently lose their bitcoin. Arguably, this creates one of the most challenging design problems within the space.
+The _[recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }})_ is the fundamental information (consisting of 12 or 24 words) used to construct a wallet. This can be encrypted and automatically backed up to the cloud, or manually backed up on paper by the user.
```

Comment:
How / what is it encrypted by? The users PIN? This also isn't explored in automatic cloud backup page.

Reply:
We do mention on the Auto Cloud page that it's encrypted by the user's PIN, but I can make this more clear.

Reply:
I added more detail ([e986b82](https://github.com/BitcoinDesign/Guide/pull/892/commits/e986b82aa3dea30ba967b21828f248fdd0a66768)) on the Auto Cloud page about how the PIN is used to encrypt the recovery phrase.

Reply:
I have some ideas for ways we could encrypt the cloud backup without requiring a PIN. I'd like to mock those up, but I think I'll punt for a later PR.

Reply:
Sounds good to me, lets address that then